README.md

```
@@ -1,2 +1,1081 @@
+
+This was a public repository for discussions relating to Open Source Program Offices on GitHub.
+# Primer Primitives
+
+This repo contains values for color, spacing, and typography primitives for use with [Primer][primer], GitHub's design system.
+
+## Install
+
+This repository is distributed on [npm][npm]. After [installing npm][install-npm], you can install `@primer/primitives` with this command.
+
+```sh
+$ npm install --save @primer/primitives
```

markdownlint-cli2 (0.18.1)

Dollar signs used before commands without showing output

(MD014, commands-show-output)